packages/ng-primitives/progress/src/progress-label/progress-label.ts

```diff
@@ -6,6 +6,8 @@ import { injectProgressState } from '../progress/progress-state';
   selector: '[ngpProgressLabel]',
   exportAs: 'ngpProgressLabel',
   host: {
+    '[attr.id]': 'id()',
+    '[attr.for]': 'state().id()',
```

Comment:
Could we possibly add a check here to see if the element is a label element and only add the for attribute when it is? The selector technically allows any element, and people may be using other elements and the for attribute is only valid on labels.